Source/SimCacheModule/Source/Subsystems/AircraftTracker/AircraftTracker.h

```diff
@@ -33,10 +34,12 @@ class AircraftTracker
 
 private:
 
-	Utils::NativeEventDispatcher&	InternalEventDispatcher;
-	SimConnect::ISimConnectClient&	SimConnectClient;
+	Utils::NativeEventDispatcher&				InternalEventDispatcher;
+	SimConnect::ISimConnectClient&				SimConnectClient;
 
-	SimConnect::Handle				UserAircraftPositionUpdatedHandle;
+	SimConnect::Handle							UserAircraftPositionUpdatedHandle;
+
+	Utils::Optional< Utils::EarthCoordinate >	LastKnownPosition;
```

Comment:
Do we want a public getter for `LastKnownPosition` as well (e.g. in case we want to access the location from something that doesn't subscribe to the `AircraftPositionUpdatedEvent`)?

Also, if we modify the aircraft tracker subsystem so it's not tracking the aircraft _at all times_, do we want to have some measure of how stale the location is?  Or do we just null out the optional if the tracker isn't currently "engaged" (i.e. basically say that if it's present, it's likely to be reasonably current)?

Reply:
Added the getter - totally forgot to do that! I think your other point about knowing more context about the last known position is unknown at the moment, since our only current behaviour is to update it at all times. If we do ever switch updates on/off I'd imagine we would null out the last known position once the updates stop.